src/sass/_settings.scss

```diff
@@ -70,7 +70,7 @@ $author-list-large-max: 10;
 $author-list-medium-max: 3;
 $author-list-small-max: 3;
 
-$institution-list-extra-large-max: 3;
-$institution-list-large-max: 3;
-$institution-list-medium-max: 2;
-$institution-list-small-max: 2;
+$affiliation-list-extra-large-max: 3;
```

Comment:
Any reason for this to change to affilation as on Journal these are known as institutions. Has the naming of these changed over the years?

Reply:
Spoke with Dave. This is Encoda's terminology. We could change all SASS to institutions, but encoda will still be giving us affiliations, and we still have to map the terms. We agreed to keep affiliations in server for the short term, and we'll map affiliation -> institutions on the client when the time comes. Hopefully, we will then also at some point remove encoda and be able to map all the way back to source institutions.